### DISCUSSION TITLE
Lux app displaying half of actual values

### DISCUSSION THREAD

**Discussion by sabelosimelane** [celsworth/lxp-bridge#169] [General]
I am not sure if I am the only one experiencing this issue but perhaps I'm not.

I have a 5Kw Hybrid inverter and I have set it to hybrid mode. I have a water heater with a separate power meter that allows me to see how much power the heater is drawing. The meter is showing that the draw is 2.7kw but the mobile app is showing that the consumption is 1.4kw.

 The effect of this is that the inverter is not using the maximum power output from the PV array which should be 3.6kw. So, I find that the inverter draws 1.4kw from the array and draws the rest from the grid.

Should it not draw the maximum power from the PV array then supplement the balance with the grid? Am I missing a config perhaps?

![Screenshot_20230603_114204_LuxPowerView](https://github.com/celsworth/lxp-bridge/assets/1929271/3d955860-f85a-4bf2-a0d8-12178f763dad)

![Screenshot 2023-06-03 at 11 43 50](https://github.com/celsworth/lxp-bridge/assets/1929271/53ecf314-9a12-4e82-9c36-fde835f81b73)

**Comment by sabelosimelane** [comment]
By the way, the lxp-bridge is also returning the same values as the mobile app which I am displaying on Home Assistant.

**Comment by celsworth** [comment]
Sorry, no clue how to advise on this one - but just for confirmation this seems to be a problem with the inverter/install itself measuring current incorrectly? Has it done this since installation or recently changed? This is probably the sort of thing it's worth asking your installer about.

lxp-bridge wouldn't change anything (without you explicitly setting a register) that could affect this, and to be honest I'm not even sure if there is a register you could set that would do it anyway.

It's almost like the wrong CT is plugged in (different winding ratio perhaps) but on my inverter the CTs have different sockets (RJ45 and RJ11 I think) so it's impossible to get them swapped around.

Sorry couldn't be more help :(
